etc/ecs_schemas/1.4.0/ecs_flat.json

```diff
@@ -1390,7 +1390,7 @@
         "name": "intrusion_detection"
       },
       {
-        "description": "Malware detection events and alerts. Use this category to visualize and analyze malware detections from EDR/EPP systems such as Elastic Endpoint Security, Symantec Endpoint Protection, Crowdstrike, and network IDS/IPS systems such as Suricata, or other sources of malware-related events such as Palo Alto Networks threat logs and Wildfire logs.\n",
+        "description": "Malware detection events and alerts. Use this category to visualize and analyze malware detections from EDR/EPP systems such as Endpoint Security, Symantec Endpoint Protection, Crowdstrike, and network IDS/IPS systems such as Suricata, or other sources of malware-related events such as Palo Alto Networks threat logs and Wildfire logs.\n",
```

Comment:
Hey @kevinlog, I would revert the changes on the ecs files as these are autogenerated based on the ECS releases and so would have no impact on naming anyway.